### DISCUSSION TITLE
The service is free, why should i install my own?

### DISCUSSION THREAD

**Discussion by jesperordrup** [uxbox/uxbox#2319] [General]
Hi all,

Youve created the oss and you've created a free online service? Is it provided as demo or whats keeping me from just using that instead of installing a local?

**Comment by niwinz** [comment]
It is better to ask it on https://community.penpot.app/

**Reply by jesperordrup** [reply]
https://community.penpot.app/t/the-service-is-free-why-should-i-install-my-own/1377
